scripts/lib.py

```diff
@@ -96,7 +96,7 @@ def get_seed(seed):
   """
   if seed >= 0:
     return seed
-  return random.getrandbits(32)
```

Comment:
Seed should be signed integer - passing 32 bits unsigned value is outside of the range and cause error.
Size was changed to 31 bits to avoid this issue.